src/model/exec-with-error-check.ts

```diff
@@ -12,18 +13,35 @@ export async function execWithErrorCheck(
     return result.exitCode;
   }
 
-  // Check for errors in the Build Results section
-  const match = result.stdout.match(/^#\s*Build results\s*#(.*)^Size:/ms);
+  // Check for errors in the Unity Build Results section
+  const unityMatch = result.stdout.match(/^#\s*Build results\s*#(.*)^Size:/ms);
 
-  if (match) {
-    const buildResults = match[1];
+  if (unityMatch) {
+    const buildResults = unityMatch[1];
     const errorMatch = buildResults.match(/^Errors:\s*(\d+)$/m);
     if (errorMatch && Number.parseInt(errorMatch[1], 10) !== 0) {
       throw new Error(`There was an error building the project. Please read the logs for details.`);
     }
-  } else {
-    throw new Error(`There was an error building the project. Please read the logs for details.`);
+    await summary.addHeading('Build Results').addQuote(unityMatch[0]).write();
```

Comment:
Should this be logged before the exception? Otherwise you wouldn't see how many errors/exceptions there are, if you have any.

Reply:
Yes, it should.